addon/components/frost-action-bar.js

```diff
@@ -80,25 +93,162 @@ export default Component.extend({
   },
 
   @readOnly
-  @computed('controls')
-  isControlsObject (controls) {
-    return typeOf(controls) === 'object'
+  @computed('controls', 'selectedItems.[]', 'moreActions')
+  /**
+   * gets controls for the action bar
```

Comment:
explain here the manipulation being done to the `controls` array with respect to 'button slicing` or preparation for creating the `More...` button if `moreActions` is true